technical-reports/resolver/rationale.md

```diff
@@ -1,48 +0,0 @@
-## Rationale
-
-### Context and Problem Statement
-
-In design systems, especially those at scale, managing multiple themes, modes, brands, and other modifiers can become complex. Designers and developers often struggle with:
-
-- **Naming Conflicts**: Using complex naming conventions to differentiate tokens across different themes or modes, leading to brittle and hard-to-maintain token names.
-- **Scalability Issues**: Difficulty in scaling token management when new dimensions are added, such as additional themes or brands.
-- **Tooling Limitations**: Existing tools may not support multi-dimensional token resolution, forcing teams to create workarounds that complicate their workflows.
-- **Separation of Concerns**: Embedding resolution logic within token files mixes data with behavior, violating separation of
-  concerns.
-
-### Parameters
```

Comment:
Removed because these are all **design goals,** not the specification itself. These may all be _motivators_ behind writing a specification! But writing down motivations alone is irrelevant to a specification.